it/uk/gov/hmrc/traderservices/controllers/TraderServicesFrontendISpec.scala

```diff
@@ -1004,7 +1004,123 @@ class TraderServicesFrontendISpec
         )
       }
     }
+  }
 
+  "GET /pre-clearance/import-questions/vessel-info" should {
```

Comment:
These tests are fine, just need to change the C1601 in some of the previous ones (for the optional export page)